test/tests.js

```diff
@@ -712,7 +712,7 @@ describe( 'passport-saml /', function() {
     it( 'should throw an error if cert property is provided to saml constructor but is empty', function() {
       should(function() {
         new SAML( { cert: null } );
-      }).throw('Invalid property: cert must not be empty');
+      }).throw({ message: 'Invalid property: cert must not be empty' });
```

Comment:
Based on [this](http://shouldjs.github.io/#assertion-throw), it doesn't seem that we should haven an object here and that the previous syntax should continue to work.

Reply:
True, the extra wrapper was unnecessary! I've removed it